configs/vision_transformer/vit-base-p16_ft-4xb544_in1k-224_ipu.py

```diff
@@ -0,0 +1,119 @@
+_base_ = [
+    '../_base_/models/vit-base-p16.py',
+    '../_base_/datasets/imagenet_bs64_pil_resize_autoaug.py',
+    '../_base_/schedules/imagenet_bs4096_AdamW.py',
```

Comment:
Looks almost all settings in the `_base_/schedules/imagenet_bs4096_AdamW.py` is overrided, just remove the base config.